deployment/docker/README.md

```diff
@@ -189,6 +189,37 @@ Please see more examples on integration of VictoriaLogs with other log shippers
 * [fluentd](https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/deployment/docker/victorialogs/fluentd)
 * [datadog-serverless](https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/deployment/docker/victorialogs/datadog-serverless)
 
+## VictoriaTraces server
+
+To spin-up environment with VictoriaTraces run the following command:
+```
+make docker-vt-single-up
+```
+_See [compose-vt-single.yml](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/deployment/docker/compose-vt-single.yml)_
+
+VictoriaTraces will be accessible on the `--httpListenAddr=:9428` port.
+In addition to VictoriaTraces server, the docker compose contains the following components:
+* [HotROD](https://hub.docker.com/r/jaegertracing/example-hotrod) application to generate trace data.
+* `VictoriaMetrics single-node` to collect metrics from all the components;
+* [Grafana](#grafana) is configured with [VictoriaLogs datasource](https://github.com/VictoriaMetrics/victorialogs-datasource).
+* [vmalert](#vmalert) is configured to query `VictoriaLogs single-node`, and send alerts state
+  and recording rules results to `VictoriaMetrics single-node`;
+* [alertmanager](#alertmanager) is configured to receive notifications from `vmalert`.
+
+<img alt="VictoriaLogs single-server deployment" width="500" src="assets/vl-single-server.png">
+
+To access HotROD, use link [http://localhost:8080](http://localhost:8080).
```

Comment:
We should mention that in order to get some demo traces, one should open hotROD and click some links around.

Reply:
Updated